src/views/v2/tickets/notice.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Block: Tickets
+ * Notice
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/tickets/notice.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1amp
+ *
+ * @since TBD
+ * @version TBD
+ */
+
+$this->template(
+	'components/notice',
+	[
+		'id'              => 'tribe-tickets__notice__tickets-in-cart',
+		'notice_classes'  => [
+			'tribe-tickets__notice--barred',
+			'tribe-tickets__notice--barred-left',
+		],
+		'content_classes' => [
+			'tribe-common-b3',
+		],
+		'content'         => __( 'The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.', 'event-tickets' ),
+	]
+);
```

Comment:
Thinking of the type of call, which is a notice, I preferred to make it an independent template. We already have this type of thing in other places. And in case somebody wants to add a class to the notice (quite likely when customizing) they don't have to review the template changes whenever we make changes/update the parent template (most likely than updating the notice).